LethalAPI.Core/LethalAPI.Core.csproj

```diff
@@ -13,48 +13,52 @@
             https://api.nuget.org/v3/index.json;
             https://nuget.bepinex.dev/v3/index.json
         </RestoreAdditionalProjectSources>
-        <DocumentationFile>..\bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
-        <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)LethalAPI.ruleset</CodeAnalysisRuleSet>
-        <OutputPath>..\bin\$(Configuration)\</OutputPath>
+        <DocumentationFile>..\bin\Debug\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+        <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\LethalAPI.ruleset</CodeAnalysisRuleSet>
+        <OutputPath>..\bin\Debug\</OutputPath>
```

Comment:
Output should remain \bin\, not \bin\Debug?

Reply:
There are 4 different build options for the bootstrapper, but they are solution based. This ensures that the output is to a single folder, which helps prevent clutter and confusion. It also ensures the core dll is always in the same spot preventing old unbuilt dlls from accidentally being used. The actual assembly name does change with the configuration to prevent problems.

Sidenote:
- The Debug build option should not be released as it will always throw type load exceptions. It does allow us to check all 3 bootstrapper precompiler variables without having to do each one independently (ie 3x faster). Its primarily to ensure no warnings or errors are present in the build.

Output without this change:
```
bin/
  ~/Debug/net48/
    ~/LethalBootstrapper.Debug.dll
    ~/LethalAPI.Core.dll
  ~/MelonLoader/net48/
    ~/LethalBootstrapper.MelonLoader.dll
    ~/LethalAPI.Core.dll
  ~/BepInEx/net48/
    ~/LethalBootstrapper.BepInEx.dll
    ~/LethalAPI.Core.dll
  ~/Manual/net48/
    ~/LethalBootstrapper.Manual.dll
    ~/LethalAPI.Core.dll
```
vs Output with change:
```
  ~/Debug/net48/
    ~/LethalBootstrapper.Debug.dll
    ~/LethalBootstrapper.Manual.dll
    ~/LethalBootstrapper.BepInEx.dll
    ~/LethalBootstrapper.MelonLoader.dll
    ~/LethalAPI.Core.dll
```

Reply:
Right, but then if you build for release, it becomes ~/Debug/Release/...?

Instead, it'd be better if the different bootstrapper artifacts were placed in the correct build type tbh. Basically output 2, but without *forcing* Debug.

Reply:
In theory debug should only be for the dotnet test command, to ensure there aren't warnings or errors before proceeding with the three separate builds.

Reply:
I was considering renaming it to release which is probably the right decision, but at the time I didn't want to break my previous update scripts

Reply:
I will rename to Release to make it more consistent. The Debug solution will still build to the release but again, it shouldn't be used for the loader at all, and only devs will be able to access it anyways.